content/blog/blog-post-08.md

```diff
@@ -6,15 +6,16 @@ tags: ["blog"]
 thumbnail: img/blog/2024/kokkos-hpsf.png
 ---
 
-Kokkos is among the six initial technical projets of the newly created High Performance Software Foundation ([HPSF](https://hpsf.org)).
+Kokkos is among the six initial technical projects of the newly created High Performance Software Foundation ([HPSF](https://hpsf.org)).
 HPSF is set up as an umbrella organization within the Linux Foundation to foster collaboration around the development and support of its associated technical projects.
-Conceived through an initial collaboration between the Spack and the Kokkos team, this effort has brought together institutions that want to foster a more collaborative effort of developing and maintaining key open-source projects in the space of high performance computing and related compute intensive areas.
+Conceived through an initial collaboration between the Spack and the Kokkos team, this effort has brought together institutions that want to foster a more cooperative effort of developing and maintaining key open-source projects in the space of high performance computing and related compute intensive areas.
 
 Historically the HPC community has been characterized by a large amount of fragmentation, with many similar or even identical capabilities developed at different institutions.
-The lack of reliable software support funding mechanism in the scientific community led to a reluctance of relying on third party libraries, and the lack of community governance models hindered the establishment of joint multi-institutional development efforts.
+Many projects were reluctant to rely on third party libraries due to the lack of reliable software support funding mechanism in the scientific community.
```

Comment:
Minor suggestion:  "...rely on third - party libraries, because their long-term development, stability and availability were uncertain."